src/App.tsx

```diff
@@ -43,7 +43,7 @@ const ImployeeListItem = (props: {item: Imployee}) => {
   return (
     <ImployeeContainer>
       <div>{props.item.id}</div>
-      <div>{props.item.firstName}</div>
+      <div><a href='/employee-details'>{props.item.firstName}</a></div>
```

Comment:
This is just temporary but for now it would be best if we could use `Link`. instead of an `anchor` tag. (not a huge deal)